include/titan/options.h

```diff
@@ -161,6 +161,18 @@ struct TitanCFOptions : public ColumnFamilyOptions {
   // Default: false
   bool skip_value_in_compaction_filter{false};
 
+  // The size of each block in blob file. 0 means not using block-based mode,
```

Comment:
Curious how we usually test changes on Titan. Do we rely on tests in TiKV?

Reply:
Added a test case